unable to compile with openvino support

I downloaded a fresh clone of the repo, and am unable to compile for openvino support. The plain `make` version compiles, and compiling with openblas support also works.

Pop_OS 22.04
OpenVino 2024.2.0

Below is the outputs

(openvino_conv_env) bbballs@pop-os:~/git/whisper.cpp$ cmake -B build -DWHISPER_OPENVINO=1
-- The C compiler identification is GNU 11.4.0
-- The CXX compiler identification is GNU 11.4.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found Git: /usr/bin/git (found version "2.34.1") 
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found OpenMP_C: -fopenmp (found version "4.5") 
-- Found OpenMP_CXX: -fopenmp (found version "4.5") 
-- Found OpenMP: TRUE (found version "4.5")  
-- OpenMP found
-- Warning: ccache not found - consider installing it for faster compilation or disable this warning with GGML_CCACHE=OFF
-- CMAKE_SYSTEM_PROCESSOR: x86_64
-- x86 detected
-- Configuring done
-- Generating done
-- Build files have been written to: /home/bbballs/git/whisper.cpp/build

(openvino_conv_env) bbballs@pop-os:~/git/whisper.cpp$ cmake --build build -j --config Release
[  5%] Building CXX object src/CMakeFiles/whisper.openvino.dir/openvino/whisper-openvino-encoder.cpp.o
[ 15%] Building C object ggml/src/CMakeFiles/ggml.dir/ggml-alloc.c.o
[ 15%] Building C object ggml/src/CMakeFiles/ggml.dir/ggml-backend.c.o
[ 20%] Building C object ggml/src/CMakeFiles/ggml.dir/ggml.c.o
[ 25%] Building C object ggml/src/CMakeFiles/ggml.dir/ggml-quants.c.o
/home/bbballs/git/whisper.cpp_2/src/openvino/whisper-openvino-encoder.cpp:2:10: fatal error: ggml.h: No such file or directory
    2 | #include "ggml.h"
      |          ^~~~~~~~
compilation terminated.
gmake[2]: *** [src/CMakeFiles/whisper.openvino.dir/build.make:76: src/CMakeFiles/whisper.openvino.dir/openvino/whisper-openvino-encoder.cpp.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:248: src/CMakeFiles/whisper.openvino.dir/all] Error 2
gmake[1]: *** Waiting for unfinished jobs....
[ 30%] Linking C shared library libggml.so
[ 30%] Built target ggml
gmake: *** [Makefile:136: all] Error 2

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -123,6 +123,10 @@ set_target_properties(whisper PROPERTIES
 target_include_directories(whisper PUBLIC . ../include)
 target_compile_features   (whisper PUBLIC cxx_std_11) # don't bump
 
+if (WHISPER_EXTRA_FLAGS)
+    target_compile_options(whisper PRIVATE ${WHISPER_EXTRA_FLAGS})
+endif()
+
 target_link_libraries(whisper PUBLIC ggml)
 
 if (WHISPER_COREML)

